### ISSUE TITLE
A feature to lable links and graphics where NVDA cannot.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3333]
**Reported by HariC on 2013-07-05 02:04**
I would like to see a built-in feature in NVDA where we can lable links and graphics ourselves if the NVDA cannot make any reading of these. Let's say we found a link/button on the web and have determined that it'sfor "print", but the NVDA could not read it; we can then apply labling feature of NVDA to assign a name for it. Just like in System Access/JAWS, we could have NVDA try to detect the relevant tag/lable, if not, then we could simply issue a command, such as shift+NVDA-key+G, to lable it.

**Comment by leonardder** [comment]
Closing n favour of newer duplicate #5874

**Comment by leonardder** [comment]
See also #7194 and #6619
